driver-core/src/main/java/com/datastax/driver/core/policies/DCAwareRoundRobinPolicy.java

```diff
@@ -395,24 +354,10 @@ public void onUp(Host host) {
                 return;
         }
         dcHosts.addIfAbsent(host);
-
-        CopyOnWriteArrayList<Host> dcSuspected = perDcSuspectedHosts.get(dc(host));
-        if (dcSuspected != null)
-            dcSuspected.remove(host);
     }
 
     @Override
     public void onSuspected(Host host) {
```

Comment:
Oh sorry, just saw your comment later on in the code :), so this is being deprecated.